.github/workflows/codeforafrica-deploy-review-app.yml

```diff
@@ -0,0 +1,135 @@
+          target: ${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ secrets.CODEFORAFRICA_MONGO_DB_NAME }}-${{github.event.pull_request.number}}
+
+      - name: Build Docker image
+        uses: docker/build-push-action@v3
+        with:
+          build-args: |
+            MONGODB_URL=${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ secrets.CODEFORAFRICA_MONGO_DB_NAME }}-${{github.event.pull_request.number}}
+            NEXT_PUBLIC_APP_URL=${{ env.NEXT_PUBLIC_APP_URL }}
+            PAYLOAD_SECRET=${{ secrets.CODEFORAFRICA_PAYLOAD_SECRET_KEY }}
+            GHOST_URL=${{ secrets.GHOST_URL }}
+            GHOST_API_KEY=${{ secrets.GHOST_API_KEY }}
+            NEXT_PUBLIC_APP_LOGO_URL=${{ secrets.NEXT_PUBLIC_APP_CFA_LOGO_URL }}
+            NEXT_PUBLIC_APP_NAME=${{ secrets.NEXT_PUBLIC_APP_CFA_NAME }}
+            VERCEL_URL=https://codeforafrica.vercel.app
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new
+          context: .
+          file: ./Dockerfile.codeforafrica
+          push: true
+          tags: "${{ env.IMAGE_NAME }}:${{ github.sha }}"
+
+      - name: Push to dokku
+        uses: dokku/github-action@master
+        with:
+          # create a review app
+          command: review-apps:create
+          git_remote_url: ${{ env.DOKKU_REMOTE_URL }}/codeforafrica-dev
+          # specify a name for the review app
+          review_app_name: codeforafrica-review-${{ github.event.pull_request.number }}
+          ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
+          deploy_docker_image: ${{ env.IMAGE_NAME }}:${{ github.sha }}
+
+      - name: Set Review App Domain Name
+        uses: appleboy/ssh-action@v1.0.0
+        with:
+          host: ui-1.dev.codeforafrica.org
+          key: ${{ secrets.SSH_PRIVATE_KEY }}
+          username: azureuser
+          script: |
+            dokku domains:add codeforafrica-review-${{github.event.pull_request.number}} codeforafrica-review-${{github.event.pull_request.number}}.cms.review.codeforafrica.org --no-restart
+            dokku config:set codeforafrica-review-${{github.event.pull_request.number}} PAYLOAD_PUBLIC_APP_URL=http://codeforafrica-review-${{github.event.pull_request.number}}.cms.review.codeforafrica.org --no-restart
+            dokku config:set codeforafrica-review-${{github.event.pull_request.number}} MONGODB_URL=${{ secrets.CODEFORAFRICA_MONGO_URL }}/${{ secrets.CODEFORAFRICA_MONGO_DB_NAME }}-${{github.event.pull_request.number}} --no-restart
+
+      - name: Update Preview URL
+        uses: nefrob/pr-description@v1.0.0
+        with:
+          content: "[Preview URL](http://codeforafrica-review-${{ github.event.pull_request.number }}.cms.review.codeforafrica.org)"
+          regex: "<!-- preview url start regex match -->.*?<!-- preview url end regex match -->"
+          regexFlags: ims
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Since we can use GitHub CLI in a [workflow](https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows), I think we can just [comment/update last comment](https://cli.github.com/manual/gh_pr_comment). I'm wary of us including and using "less" known actions.